WeightProducer/python/MCSampleValues.py

```diff
@@ -251,669 +247,501 @@ class MCSampleValuesHelper():
-                kFactor_2017=1.162, kFactorSource_2017="XSDB NNLO/LO=61526.7/52940",
-                kFactor_2018=1.164, kFactorSource_2018="XSDB NNLO/LO=61526.7/52850",
+        },
+        "TTToSemiLeptonic_mtop173p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=668.6, XSSource_13TeV="XSDB (NLO)",
             ),
-            "Correction" : CorrValues(
-                Corr_2017=4.200, CorrSource_2017="https://twiki.cern.ch/twiki/bin/viewauth/CMS/MCKnownIssues#WJetsToLNu_HT_and_DYJets_HT_LO_M",
-                Corr_2018=4.200, CorrSource_2018="Same as 2017",
+        },
+        "TTToSemiLeptonic_mtop175p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=633.4, XSSource_13TeV="XSDB (NLO)",
             ),
         },
-        "WJetsToLNu" : {
-            "CrossSection" : XSValues(
-                XS_13TeV=61526.7, XSSource_13TeV="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#W_jets NNLO (60430.0 @ NLO)",
-                XS_2016=50260.0, XSSource_2016="XSDB (LO)",
-                XS_2017=52940.0, XSSource_2017="XSDB (LO)",
-                XS_2018=52850.0, XSSource_2018="XSDB (LO)",
+        "TTToSemiLeptonic_mtop178p5" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=584.6, XSSource_13TeV="XSDB (NLO)",
             ),
         },
-        "WJetsToQQ_HT400to600_qc19_3j" : {
+        "TTGamma_SingleLept" : {
             "CrossSection" : XSValues(
-                XS_2017=315.2, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=314.6, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=5.085, XSSource_13TeV="XSDB (LO)",
             ),
         },
-        "WJetsToQQ_HT600to800_qc19_3j" : {
+        "TTGamma_Hadronic" : {
             "CrossSection" : XSValues(
-                XS_2017=68.61, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=68.58, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=4.178, XSSource_13TeV="XSDB (LO)",
             ),
         },
-        "WJetsToQQ_HT-800toInf_qc19_3j" : {
+        "TTGamma_Dilept" : {
             "CrossSection" : XSValues(
-                XS_2017=34.71, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=34.74, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=1.502, XSSource_13TeV="XSDB (LO)",
             ),
         },
-        "WJetsToQQ_HT-600ToInf" : {
-            "CrossSection" : XSValues(
-                XS_2016=99.65, XSSource_2016="XSDB (LO)"
+        "WJetsToLNu" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=53870.0, XSSource_13TeV="XSDB (LO)",
             ),
         },
-        "QCD_HT200to300" : {
+        "WJetsToLNu_HT-70To100" : {
             "CrossSection" : XSValues(
-                XS_2016=1710000.0, XSSource_2016="XSDB (LO)",
-                XS_2017=1547000.0, XSSource_2017="XSDB (LO)",
-                XS_2018=1557000.0, XSSource_2018="XSDB (LO)",
+                XS_13TeV=1264.0, XSSource_13TeV="XSDB (LO)",
```

Comment:
Same for all three years? This is the last time I make this comment until I hear whether or not that's true in general.

Reply:
I think all years should be the same now that the PDFs are the same (but it would be good to confirm this).

Reply:
See reply to TTGamma post.